### DISCUSSION TITLE
蓝灯客户端无法直接升级，请弄一个专属支付链接！

### DISCUSSION THREAD

**Discussion by Honeydukes3** [getlantern/lantern-forum-cn#968] [General]
蓝灯客户端无法直接升级，请弄一个专属支付链接！
账号：771324905donald@tutanota.com

**Comment by MaxMKMax** [comment]
@Honeydukes3 OK没问题. 谢谢您的大力支持蓝灯!

**Comment by greySkiesMarsh** [comment]
@Honeydukes3 Hi! Please try this [link ]( https://buy.stripe.com/4gwaFe3k79C62GscMQ)to upgrade.  

I've added a month of Pro to your account just in case you still run into issues. Let me know if you still aren't able to purchase through that link and I'll see what I can do.
